### ISSUE TITLE
games/apps are fps dependent

### ISSUE THREAD

**Issue by Trickiy** [Wolfyxon/3ds-web-stuff#71]
Everything is fps dependent meaning when the fps is low the physics is slowed down but when the fps is high like 60 or more the physics is sped up a lot. Physics should be the same regaurdless of framerate allowing fps and physics to be completely independent.

**Comment by Wolfyxon** [comment]
> What should happen is when the fps is lowered the physics get sped up but when the fps increases the physics should slow down

Yes, that's what happens (or what should). All dynamic games are specifically designed to work in the speed no matter the FPS by taking time from the previous frame then calculating the offset (delta) then multiplying all the speed values.
Example:
```js
const speed = 10;
var prevFrameTime = Date.now();

setInterval(function() {
    const delta = (Date.now() - prevFrameTime);
    prevFrameTime = Date.now();
   
   if(isBtnPressed("right") player.moveXY(speed * delta, 0);
   if(isBtnPressed("left") player.moveXY(-speed * delta, 0);
});
```

I tested many games and forced them to run at lower FPS, but couldn't replicate the issue, they run at the same speed.

> This is very apparent in elementGame.js as when on new 3ds I get arrround 20 pigeons but on old I barely get 10.

Pigeon spawning doesn't use the delta system, but it's just for testing so I don't think it's that important.
